examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  selector:
+    component: elasticsearch
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        image: quay.io/pires/docker-elasticsearch-kubernetes:1.7.1-4
+        env:
+        - name: KUBERNETES_TRUST_CERT
+          value: "true"
```

Comment:
The `KUBERNETES_SERVICE_HOST` works on the `default` namespace alone? Can one app in a different namespace get there?

Also, we need a new release of the discovery plug-in @jimmidyson?